target/linux/ipq40xx/base-files/etc/board.d/01_leds

```diff
@@ -98,6 +98,13 @@ netgear,ex6150v2)
 	ucidef_set_led_wlan "wlan2g" "WLAN2G" "green:router" "phy0tpt"
 	ucidef_set_led_wlan "wlan5g" "WLAN5G" "green:client" "phy1tpt"
 	;;
+netgear,ex6200v2)
+        ucidef_set_led_wlan "wlan2g" "WLAN2G" "green:wlan2g" "phy0radio"
+        ucidef_set_led_wlan "wlan5g" "WLAN5G" "green:wlan5g" "phy1radio"
+        ucidef_set_led_usbport "usb" "USB" "green:usb" "usbport" "usbdev" "usb4" "usb3" "usb2" "usb1" "usb3-port1" "usb2-port1" "usb1-port1"
```

Comment:
USB LED-s can be triggered via DTS

Reply:
is this a requirement or a suggestion? I jest but in all seriousness, I'm tripping up hard on this. 

Based on other commits, I've been trying to make use linux, default-trigger. I've tried setting it to "usbport" or "usbdev" but no dice. 

I did figure out which trigger in 01_leds actually was the correct trigger and tried entering that, which is what is below but still no dice. Any suggestions? I also made sure to include kmod-usb-ledtrig-usbport in my test build for reference. 

 `usb_green {
                        label = "green:usb";
                        gpios = <&led_gpio 0 GPIO_ACTIVE_LOW>;
                        linux,default-trigger = "usb2-port1";
                        };`

Reply:
Sorry, I completely missed the email notification.

Check out the EA8300 DTS in IPQ40xx, its implementing USB triggering via DTS

Reply:
thanks for the tip!

I did a little digging & comparing and found that the following is missing in the 4019.dtsi that I included, adding this in my dtsi successfully implements USB triggering. 

`                usb2@60f8800 {
                        dwc3@6000000 {
                                usb2_port1: port@1 {
                                        reg = <1>;
                                        #trigger-source-cells = <0>;
                                };
                        };
                };

                usb3@8af8800 {
                        dwc3@8a00000 {
                                usb3_port1: port@1 {
                                        reg = <1>;
                                        #trigger-source-cells = <0>;
                                };
                        };
                };
`

I do have a formatting question though before I update my PR; the 4019.dtsi defines labels for usb2 and usb3 but when I used to those in my code (see below), I get syntax errors when compiling. Simply replacing the labels with what's above resolves the errors. Am I okay to not use the labels? if not, is there an obvious error I'm missing?

` &usb2 {
                        dwc3@6000000 {
                                usb2_port1: port@1 {
                                        reg = <1>;
                                        #trigger-source-cells = <0>;
                                };
                        };
                };

                &usb3 {
                        dwc3@8a00000 {
                                usb3_port1: port@1 {
                                        reg = <1>;
                                        #trigger-source-cells = <0>;
                                };
                        };
                };
`


![image](https://user-images.githubusercontent.com/86842761/222964474-1a23e4fc-31ec-4b3c-a937-e8e52c1f8c8e.png)

Reply:
You cannot just rename to use the labels, those nodes now need to go outside of the `soc`.
Just put them below `&cryptobam` like:

```
&cryptobam {
	status = "okay";
};

&usb2 {
	dwc3@6000000 {
		usb2_port1: port@1 {
			reg = <1>;
			#trigger-source-cells = <0>;
		};
	};
};

&usb3 {
	dwc3@8a00000 {
		usb3_port1: port@1 {
			reg = <1>;
			#trigger-source-cells = <0>;
		};
	};
};
```